cfme/tests/services/test_service_catalogs.py

```diff
@@ -24,12 +26,9 @@
 
 def pytest_generate_tests(metafunc):
     # Filter out providers without provisioning data or hosts defined
-    argnames, argvalues, idlist = testgen.infra_providers(metafunc,
-        required_fields=[
-            ['provisioning', 'template'],
-            ['provisioning', 'host'],
-            ['provisioning', 'datastore']
-        ])
+    argnames, argvalues, idlist = testgen.infra_providers(
```

Comment:
What was the rationale for changing this? The previous format I think read a little closer to yaml